pkg/store/manifestFile.go

```diff
@@ -365,47 +368,56 @@ func (s *manifestFileStore) SyncResponseStatusUpdate2(manifestId int32, failedFi
 
 // RemoveFromManifest removes files from local manifest by path.
 func (s *manifestFileStore) RemoveFromManifest(manifestId int32, removePath string) error {
+	pathLikeExpr := removePath + "%"
+	initiatedStatus := manifestFile.Local.String()
 
-	pathLikeExpr := fmt.Sprintf("'%s%%'", removePath)
-	initatedStatus := manifestFile.Local.String()
-	queryStr := fmt.Sprintf("DELETE FROM manifest_files WHERE manifest_id = %d "+
-		"AND source_path LIKE %s and status = '%s';", manifestId, pathLikeExpr, initatedStatus)
 
-	log.Debug(queryStr)
+	stmt, err := s.db.Prepare("DELETE FROM manifest_files WHERE manifest_id = ? AND source_path LIKE ? AND status = ?")
+	if err != nil {
+		log.Println("Error in RemoveFromManifest (DELETE):", err)
+		return err
+	}
+	defer stmt.Close()
 
-	_, err := s.db.Exec(queryStr)
+	_, err = stmt.Exec(manifestId, pathLikeExpr, initiatedStatus)
 	if err != nil {
+		log.Println("Error executing RemoveFromManifest (DELETE):", err)
 		return err
 	}
 
+
 	syncStatus := manifestFile.Registered.String()
 	removeStatus := manifestFile.Removed.String()
-	queryStr2 := fmt.Sprintf("UPDATE manifest_files SET status = '%s' WHERE manifest_id = %d "+
-		"AND source_path LIKE %s and status = '%s';", removeStatus, manifestId, pathLikeExpr, syncStatus)
 
-	_, err = s.db.Exec(queryStr2)
+	queryStr2 := "UPDATE manifest_files SET status = ? WHERE manifest_id = ? AND source_path LIKE ? AND status = ?"
+	_, err = s.db.Exec(queryStr2, removeStatus, manifestId, pathLikeExpr, syncStatus)
 	if err != nil {
+		log.Println("Error executing RemoveFromManifest (UPDATE):", err)
 		return err
 	}
 
 	return nil
 }
 
+
 // ResetStatusForManifest resets all files to status = LOCAL
 func (s *manifestFileStore) ResetStatusForManifest(manifestId int32) error {
 
 	currentTime := time.Now()
 
 	initiatedStatusStr := manifestFile.Local.String()
-	sqlStatement := fmt.Sprintf("UPDATE manifest_files SET status = '%s', updated_at = %d WHERE manifest_id = %d",
-		initiatedStatusStr, currentTime.Unix(), manifestId)
+	stmt, error := s.db.Prepare("UPDATE manifest_files SET status = ?, updated_at = ? WHERE manifest_id = ?")
+	if error != nil {
+		log.Println("the error is here:ResetStatusForManifest" + "here is why" + error.Error())
+		return error
+	}
 
-	log.Debug(sqlStatement)
-	// format all vals at once
-	_, err := s.db.Exec(sqlStatement)
+	defer stmt.Close()
+
+	_, err := stmt.Exec(initiatedStatusStr, currentTime.Unix(), manifestId)
 	if err != nil {
-		log.Error(err)
-		return err
+		log.Println("the error is here:ResetStatusForManifest in second if statement" + "here is why" + error.Error())
+		return error
```

Comment:
should these still reference `err` not `error`?

Reply:
yes, I fixed this typo in the newest version